.github/workflows/build_and_test.yaml

```diff
@@ -5,36 +5,22 @@ on:
     - "main"
     - "release/v*"
     paths-ignore:
-    - "**/*.png"
+    - 'docs/**'
```

Comment:
as I mentioned earlier, we should always run the workflow on a `push`, and not ignore any paths, 
the `docs` path can be ignored on a `pull_request`

Reply:
Sorry by editing the wrong fields!